src/app/(get-started)/[lng]/get-started/partials/tab-contents/TabText.tsx

```diff
@@ -1,23 +1,166 @@
+import { Button } from "@/components/ui/button"
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card"
+import { InputGroup, InputGroupInput } from "@/components/ui/input-group"
+import { ScrollArea } from "@/components/ui/scroll-area"
 import { TabsContent } from "@/components/ui/tabs"
 import { motion, Variants } from "motion/react"
+import { useOnboardingStore } from "../../store/onboarding.store"
+import { InputWithLength } from "@/components/InputWithLength"
+import { useState } from "react"
+import { calculateTextSizeFromLength } from "@/utils"
+import { IconDotsVertical } from "@tabler/icons-react"
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger
+} from "@/components/ui/dropdown-menu"
 
 interface TabTextProps {
   motionVariants: Variants
 }
 
 const TabText = ({ motionVariants }: TabTextProps) => {
+  const { textSources, setTextSources } = useOnboardingStore()
+
+  const [title, setTitle] = useState("")
+  const [description, setDescription] = useState("")
+
+  const handleDescriptionChange = (
+    e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    setDescription(e.target.value)
+  }
+
+  const handleTitleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(e.target.value)
+  }
+
+  const handleAddText = () => {
+    const size = calculateTextSizeFromLength(description).bytes
+    setTextSources([...textSources, { title, description, size }])
+    setTitle("")
+    setDescription("")
+  }
+
+  const handleDeleteText = (idx: number) => {
+    setTextSources(textSources.filter((_, i) => i !== idx))
+  }
+
   return (
-    <TabsContent value="text" className="mt-4 h-96 rounded-md bg-zinc-200">
+    <TabsContent value="text" className="mt-4 h-96 rounded-md">
       <motion.div
-        key="text-content"
+        key="website-content"
```

Comment:
```suggestion
        key="text-content"
```

in the TabWebsite.tsx the `website-content` key is used